CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+cmake_minimum_required(VERSION 3.22)
+
+#=============================
+# Project / Package Metadata
+#=============================
+project(minisketch
+  VERSION 0.0.1
+  DESCRIPTION "A library for BCH-based set reconciliation"
+  HOMEPAGE_URL "https://github.com/bitcoin-core/minisketch"
+  LANGUAGES CXX
+)
+
+# ============================================================
+# Project Initialization
+# ============================================================
+enable_testing()
```

Comment:
I believe this is superior to `include(CTest)`, as the latter has a few drawbacks.

Some other developers, namely @purpleKarrot, disagree.
See: https://github.com/bitcoin-core/libmultiprocess/pull/145